docs/visualization/visualization_api/communication_information.md

```diff
@@ -43,3 +57,7 @@ plot.show()
 ```
 
 ![communication_latency_time_line](../../imgs/communication_latency_time_line.png)
+
+The horizontal axis means `Time [s]` (changeable in ['system_time', 'sim_time', 'index']).
+The vertical axis means `Latency [ms]` from one callback publishes a topic to next callback starts.
+This API is used to confirm the communication time.
```

Comment:
@takeshi-iwanari 
I adapted your comment.
https://github.com/tier4/CARET_doc/pull/95#discussion_r999198661
> Explanation of frequency, period, and latency is needed.
> For instance, latency is the duration from oo to oo.

For example, here.
I added the explanation of each axis and what developer can confirm.